edgegrid/gtm_client_api.go

```diff
@@ -32,7 +32,7 @@ func (c *GTMClient) DomainStatus(name string) (*ResourceStatus, error) {
 	return status, err
 }
 
-// DomainCreate takes an Akamai GTM domain name and a domain type and creates
+// DomainCreate takes an Akamai GTM domain name and a domainType and creates
 // the domain via the Akamai GTM API.
 func (c *GTMClient) DomainCreate(name string, domainType string) (*DomainResponse, error) {
```

Comment:
If you can still change method names I think the method name `CreateDomain` would read better. Same goes for other methods, put the verb part first.

As for the doc comment, what about this?

```go
// CreateDomain sends a request to create a domain with the provided name and type.
// The result is returned as a DomainResponse.
```

In this case I bent the rules a bit because the second parameter *wants* to be `type` but cannot because that's a Go keyword. Naming it `domainType` avoids that problem, but the comment reads better with just "type" and the reader easily makes the connection. But the main difference is structuring the comment to emphasize what the method *does* instead of its inputs.